src/Generator/DiffGenerator.php

```diff
@@ -53,16 +51,16 @@ static function ($assetName) use ($filterExpression) {
                         $assetName = $assetName->getName();
                     }
 
-                    return preg_match($filterExpression, $assetName);
+                    return (bool) preg_match($filterExpression, $assetName);
```

Comment:
Matching the [docs ](https://www.doctrine-project.org/projects/doctrine-migrations/en/3.8/reference/generating-migrations.html#ignoring-custom-tables)